src/ai-did-you-mean-this/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
These 2 lines can be removed as we dropped support for python 2.

Reply:
Removed them.

Reply:
It seems you removed the one in ai-examples/setup.cfg

Reply:
Woops. I'll fix that soon.

Reply:
Fixed.